### PR TITLE
Add tracking for WAR's Nascent Chaos status timeouts/overwrites

### PR DESCRIPTION
**Pull request by ayyaruq**
This required a type fix for core Procs as well, so that's been rolled in. Changelog entries added for both WAR and core.

### CODE REVIEW

File: src/parser/jobs/war/modules/Infuriate.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			() => this.cooldowns.reduce('INFURIATE', INFURIATE_CDR),
+		)
+	}
+
+	override trackedProcs = [{
+		procStatus: this.data.statuses.NASCENT_CHAOS,
+		consumeActions: [this.data.actions.INNER_CHAOS, this.data.actions.CHAOTIC_CYCLONE],
+	}]
+
+	override droppedProcIcon = this.data.actions.INFURIATE.icon
```
Comment (AkairyuGestalter) [inline src/parser/jobs/war/modules/Infuriate.tsx:51]: You'll need to set `override show[Dropped/Overwrote]ProcSuggestions = true` if you want these to display. All of the default suggestions in core are hidden by default.

File: src/parser/jobs/war/modules/Infuriate.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	override droppedProcSeverityTiers = INFURIATE_SEVERITY
+	override droppedProcContent = <Trans id="war.infuriate.suggestions.dropped.content">Avoid letting <DataLink status="NASCENT_CHAOS"/> fall off without using it.</Trans>
```
Comment (AkairyuGestalter) [inline src/parser/jobs/war/modules/Infuriate.tsx]: The 'dropped nascent chaos' suggestion is duplicating one that's in `PrimalChaos.tsx`. Should the two files be merged? PrimalChaos looks like it's also essentially doing proc tracking for Primal Rend

**Review by AkairyuGestalter — DISMISSED** [review]
Couple things. One minor, one that might need some thinks

File: src/parser/jobs/war/modules/Infuriate.tsx (hunk 2 above)
Comment (ayyaruq) [reply src/parser/jobs/war/modules/Infuriate.tsx]: I'm gonna revert the changes to Infuriate, and port PrimalChaos to core Procs.

Comment (ayyaruq) [reply src/parser/jobs/war/modules/Infuriate.tsx]: Actually I lied, prefer to split these outright.

**Review by AkairyuGestalter — APPROVED** [review]

**Pull request merged**